controlyourtabs/gedit-control-your-tabs.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="addon">
+  <id>gedit-control-your-tabs</id>
```

Comment:
```suggestion
  <id>com.thingsthemselves.gedit.plugins.controlyourtabs</id>
```

Reply:
Indeed. That's right and much better. As for move it into `data/` may you move it yourself somehow or in next commit? Also i think email info better change to yours probably, or remove it at all. This looks more like distro specific thing, but i am not sure.